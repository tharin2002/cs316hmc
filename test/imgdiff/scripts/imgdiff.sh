#!/bin/bash
#-- sample small test

PHYSRAND=../imgdiff

BANKS=8
CAPACITY=2
LINKS=4
BSIZE=128
QDEPTH=64
XDEPTH=128
VAULTS=16
NRQSTS=3750
DRAMS=20

echo "Executing : $PHYSRAND -b $BANKS -c $CAPACITY -l $LINKS -m $BSIZE -n 1 -q $QDEPTH -x $XDEPTH\
	-d $DRAMS -v $VAULTS -N $NRQSTS -S 65656"

$PHYSRAND -b $BANKS -c $CAPACITY -l $LINKS -m $BSIZE -n 1 -q $QDEPTH -x $XDEPTH\
	-d $DRAMS -v $VAULTS -N $NRQSTS -S 65656
