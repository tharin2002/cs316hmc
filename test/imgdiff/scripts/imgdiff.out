HMCSIM_TRACE : 0 : XBAR_LATENCY : 0:0:0:2:0
HMCSIM_TRACE : 0 : XBAR_LATENCY : 0:0:1:2:0
HMCSIM_TRACE : 0 : XBAR_LATENCY : 0:0:2:2:0
HMCSIM_TRACE : 0 : XBAR_LATENCY : 0:0:3:2:0
HMCSIM_TRACE : 0 : XBAR_LATENCY : 0:0:8:2:0
HMCSIM_TRACE : 0 : XBAR_LATENCY : 0:0:9:2:0
HMCSIM_TRACE : 0 : XBAR_LATENCY : 0:0:10:2:0
HMCSIM_TRACE : 0 : XBAR_LATENCY : 0:0:11:2:0
HMCSIM_TRACE : 0 : BANK_CONFLICT : 0:0:0:4:0x00000001a2f85000
HMCSIM_TRACE : 0 : BANK_CONFLICT : 0:0:0:5:0x00000001a2f85400
HMCSIM_TRACE : 0 : BANK_CONFLICT : 0:2:0:3:0x00000001a2f84e00
HMCSIM_TRACE : 0 : BANK_CONFLICT : 0:2:0:4:0x00000001a2f85200
HMCSIM_TRACE : 0 : BANK_CONFLICT : 0:2:0:4:0x00000001a2f85200
HMCSIM_TRACE : 0 : BANK_CONFLICT : 0:2:0:4:0x00000001a2f85200
HMCSIM_TRACE : 0 : BANK_CONFLICT : 0:2:0:5:0x00000001a2f85600
HMCSIM_TRACE : 0 : RD64 : 0:0:0:4:0x00000001a2f85000:1
HMCSIM_TRACE : 0 : RD64 : 0:0:0:5:0x00000001a2f85400:1
HMCSIM_TRACE : 0 : RD64 : 0:2:0:3:0x00000001a2f84e00:1
HMCSIM_TRACE : 0 : RD64 : 0:2:0:4:0x00000001a2f85200:1
HMCSIM_TRACE : 0 : RD64 : 0:2:0:5:0x00000001a2f85600:1
HMCSIM_TRACE : 1 : BANK_CONFLICT : 0:2:0:4:0x00000001a2f85200
HMCSIM_TRACE : 1 : BANK_CONFLICT : 0:2:0:4:0x00000001a2f85200
HMCSIM_TRACE : 1 : RD64 : 0:0:0:4:0x00000001a2f85000:1
HMCSIM_TRACE : 1 : RD64 : 0:0:0:5:0x00000001a2f85400:1
HMCSIM_TRACE : 1 : RD64 : 0:2:0:3:0x00000001a2f84e00:1
HMCSIM_TRACE : 1 : RD64 : 0:2:0:4:0x00000001a2f85200:1
HMCSIM_TRACE : 1 : RD64 : 0:2:0:4:0x00000001a2f85200:1
HMCSIM_TRACE : 1 : RD64 : 0:2:0:5:0x00000001a2f85600:1
HMCSIM_TRACE : 2 : RD64 : 0:2:0:4:0x00000001a2f85200:1
HMCSIM_TRACE : 4 : XBAR_LATENCY : 0:0:0:2:0
HMCSIM_TRACE : 4 : XBAR_LATENCY : 0:0:1:2:0
HMCSIM_TRACE : 4 : XBAR_LATENCY : 0:0:2:2:0
HMCSIM_TRACE : 4 : XBAR_LATENCY : 0:0:3:2:0
HMCSIM_TRACE : 4 : XBAR_LATENCY : 0:0:8:2:0
HMCSIM_TRACE : 4 : XBAR_LATENCY : 0:0:9:2:0
HMCSIM_TRACE : 4 : XBAR_LATENCY : 0:0:10:2:0
HMCSIM_TRACE : 4 : XBAR_LATENCY : 0:0:11:2:0
HMCSIM_TRACE : 4 : BANK_CONFLICT : 0:0:0:5:0x0000000057703400
HMCSIM_TRACE : 4 : BANK_CONFLICT : 0:0:0:6:0x0000000057703800
HMCSIM_TRACE : 4 : BANK_CONFLICT : 0:2:0:4:0x0000000057703200
HMCSIM_TRACE : 4 : BANK_CONFLICT : 0:2:0:5:0x0000000057703600
HMCSIM_TRACE : 4 : BANK_CONFLICT : 0:2:0:5:0x0000000057703600
HMCSIM_TRACE : 4 : BANK_CONFLICT : 0:2:0:5:0x0000000057703600
HMCSIM_TRACE : 4 : BANK_CONFLICT : 0:2:0:6:0x0000000057703a00
HMCSIM_TRACE : 4 : RD64 : 0:0:0:5:0x0000000057703400:1
HMCSIM_TRACE : 4 : RD64 : 0:0:0:6:0x0000000057703800:1
HMCSIM_TRACE : 4 : RD64 : 0:2:0:4:0x0000000057703200:1
HMCSIM_TRACE : 4 : RD64 : 0:2:0:5:0x0000000057703600:1
HMCSIM_TRACE : 4 : RD64 : 0:2:0:6:0x0000000057703a00:1
HMCSIM_TRACE : 5 : BANK_CONFLICT : 0:2:0:5:0x0000000057703600
HMCSIM_TRACE : 5 : BANK_CONFLICT : 0:2:0:5:0x0000000057703600
HMCSIM_TRACE : 5 : RD64 : 0:0:0:5:0x0000000057703400:1
HMCSIM_TRACE : 5 : RD64 : 0:0:0:6:0x0000000057703800:1
HMCSIM_TRACE : 5 : RD64 : 0:2:0:4:0x0000000057703200:1
HMCSIM_TRACE : 5 : RD64 : 0:2:0:5:0x0000000057703600:1
HMCSIM_TRACE : 5 : RD64 : 0:2:0:5:0x0000000057703600:1
HMCSIM_TRACE : 5 : RD64 : 0:2:0:6:0x0000000057703a00:1
HMCSIM_TRACE : 6 : RD64 : 0:2:0:5:0x0000000057703600:1
HMCSIM_TRACE : 8 : XBAR_LATENCY : 0:0:0:2:0
HMCSIM_TRACE : 8 : XBAR_LATENCY : 0:0:1:3:0
HMCSIM_TRACE : 8 : XBAR_LATENCY : 0:1:0:2:1
HMCSIM_TRACE : 8 : XBAR_LATENCY : 0:1:1:3:1
HMCSIM_TRACE : 8 : XBAR_LATENCY : 0:1:2:0:1
HMCSIM_TRACE : 8 : XBAR_LATENCY : 0:2:1:3:2
HMCSIM_TRACE : 8 : XBAR_LATENCY : 0:2:2:0:2
HMCSIM_TRACE : 8 : XBAR_LATENCY : 0:3:0:2:3
HMCSIM_TRACE : 8 : XBAR_LATENCY : 0:3:2:0:3
HMCSIM_TRACE : 8 : WR64 : 0:0:0:5:0x00000000e96c1400:5
HMCSIM_TRACE : 8 : WR64 : 0:0:1:5:0x00000000e96c1440:5
HMCSIM_TRACE : 8 : WR64 : 0:0:2:5:0x00000000e96c1480:5
HMCSIM_TRACE : 8 : WR64 : 0:0:3:5:0x00000000e96c14c0:5
HMCSIM_TRACE : 8 : WR64 : 0:2:0:4:0x00000000e96c1200:5
HMCSIM_TRACE : 8 : WR64 : 0:2:1:4:0x00000000e96c1240:5
HMCSIM_TRACE : 8 : WR64 : 0:2:2:4:0x00000000e96c1280:5
HMCSIM_TRACE : 8 : WR64 : 0:2:3:4:0x00000000e96c12c0:5
HMCSIM_TRACE : 8 : WR64 : 0:3:0:4:0x00000000e96c1300:5
HMCSIM_TRACE : 8 : WR64 : 0:3:1:4:0x00000000e96c1340:5
HMCSIM_TRACE : 8 : WR64 : 0:3:2:4:0x00000000e96c1380:5
HMCSIM_TRACE : 8 : WR64 : 0:3:3:4:0x00000000e96c13c0:5
